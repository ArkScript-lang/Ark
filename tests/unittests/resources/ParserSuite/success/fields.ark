(let a b.c)
(mut d e.f.g)
(if (# aze
    hi.jk) l.m n.o.p)
(while q.r s.t)
(fun () u.v)
(begin x.y.z)