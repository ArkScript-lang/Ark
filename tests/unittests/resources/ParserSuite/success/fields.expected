( Keyword:Let Symbol:a ( Field Symbol:b Symbol:c ) )
( Keyword:Mut Symbol:d ( Field Symbol:e Symbol:f Symbol:g ) )
( Keyword:If ( ( Field Symbol:hi Symbol:jk ) ) ( Field Symbol:l Symbol:m ) ( Field Symbol:n Symbol:o Symbol:p ) )
( Keyword:While ( Field Symbol:q Symbol:r ) ( Field Symbol:s Symbol:t ) )
( Keyword:Fun ( ) ( Field Symbol:u Symbol:v ) )
( Keyword:Begin ( Field Symbol:x Symbol:y Symbol:z ) )
